test/templates/calendar/index.njk

```diff
@@ -1,14 +1,8 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
-{% from "../button/component.njk" import button %}
 {% from "./component.njk" import calendar %}
 
 {% block content %}
-<style>
-main {background-color: white;}
-
-</style>
-
```

Comment:
There is similar styling on the docs pages:
```css
.docs-component .fd-calendar__content {
    background: white;
}
```
It _seems_ like this should just be part of the calendar styling.  Without it, there is some bleed through from a non-white background because of the rounded corners.

Reply:
So I thought this at first too, but that isn't what matches the spec on the wiki page. This component needs to be updated in general, but you should be able to see the rounded corners. I left the hardcoded value in the docs site to prevent confusion until we update the styles: https://wiki.wdf.sap.corp/wiki/display/visualcore/Calendar+%28Fiori+3%29+-+Date+Picker

Reply:
Now that I look at the specs again, I don't know anymore. I think it is a white background, but the numbers shouldn't be white?

Reply:
After discussing offline with @greg-a-smith, we are going to add the white background to the scss file for now. This component will need to be updated with the newest design spec in the future.